LLVM/Clang 11 automatically adds "-Wundef-prefix=TARGET_OS_" as default compiler
argument and there are several instances in the ChemDraw code where things are not defined
--- External/ChemDraw/CMakeLists.txt.orig	2026-02-03 21:41:12
+++ External/ChemDraw/CMakeLists.txt	2026-02-03 21:41:46
@@ -119,7 +119,7 @@
   endif()

   if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-comment -Wno-parentheses -Wno-logical-op-parentheses -Wno-pointer-bool-conversion -Wno-unused-value -Wno-unsequenced -Wno-constant-logical-operand")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-comment -Wno-parentheses -Wno-logical-op-parentheses -Wno-pointer-bool-conversion -Wno-unused-value -Wno-unsequenced -Wno-constant-logical-operand -Wno-undef-prefix")
   endif()

   if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
